node-src/errorMonitoring.ts

```diff
@@ -0,0 +1,12 @@
+import * as Sentry from '@sentry/node';
+
+export default (environment: string) => {
+  Sentry.init({
```

Comment:
Did we want to include the CLI version in the [initial scope](https://docs.sentry.io/platforms/javascript/guides/node/configuration/options/#initial-scope)?

Reply:
Yes, we should, I'm just not positive how to extract that, since we pick that up by reading `package.json` once we're already into execution: https://github.com/chromaui/chromatic-cli/blob/2f184116a5aacd131ea9d8426fded57690aac314/node-src/index.ts#L98

Reply:
Apparently [process.env.npm_package_version](https://docs.npmjs.com/cli/v6/using-npm/scripts#packagejson-vars) is a thing and spits out that version value!

![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExcDh5eXAwbXhkNGRrM3gwNjB6aDRtY3NjazQwbmo5azhlMGpmbW9xMSZlcD12MV9naWZzX3NlYXJjaCZjdD1n/WfBZwNA6XSjphkYkzN/giphy.gif)

Reply:
Perfect, I'll give that a whirl.